src/js/providers/html5.js

```diff
@@ -186,6 +190,14 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
             const { video } = _this;
             const error = video.error;
             const errorCode = (error && error.code) || -1;
+
+            if (errorCode === 3 && _this.currentTime !== -1 && OS.iOS) {
+                // Workaround iOS bug https://bugs.webkit.org/show_bug.cgi?id=195452
+                _videotag.load();
+                _this.seek(_this.currentTime);
```

Comment:
with pause after seek does this always pause when it comes back into the foreground?

Reply:
The video tag remains in the paused or playing state until a call to play or pause changes that state. `load()` should not result in a playback change. I did notice this code which should have been removed with the "remain paused after seek" changes:

https://github.com/jwplayer/jwplayer/blob/e35cf79f372b0274a59c7ae23b46b2aa439a5551/src/js/providers/html5.js#L664-L668